client/src/lifecycleManager.js

```diff
@@ -427,6 +427,33 @@ class LifecycleManager extends LuigiClientBase {
     return this.currentContext.pathParams;
   }
 
+  /**
+   * Read search query parameters which are send from Luigi core
+   * @memberof Lifecycle
+   * @returns core search query parameters
+   * @example
+   * LuigiClient.getCoreSearchParams();
+   */
+  getCoreSearchParams() {
+    return this.currentContext.searchParams;
+  }
+
+  /**
+   * Sends search query parameters to Luigi core. If it is allowed on node level it will be added to url.
```

Comment:
```suggestion
   * Sends search query parameters to Luigi Core. If they are allowed on node level, the search parameters will be added to the URL.
```

I assume it's "If search parameters are allowed, they will be added to the URL", so that's why I changed it to plural, but feel free to change it if it's something else.